pynestml/utils/model_installer.py

```diff
@@ -61,23 +73,31 @@ def install_nest(target_path: str, nest_path: str) -> None:
     else:
         shell = False
 
+    # remove CMakeCache.txt if exists
+    cmakeCache = os.path.join(target_path, "CMakeCache.txt")
+    if os.path.exists(cmakeCache):
+        os.remove(cmakeCache)
+
     # first call cmake with all the arguments
     try:
-        result = subprocess.check_call(cmake_cmd, stderr=subprocess.STDOUT, shell=shell,
-                                       cwd=str(os.path.join(target_path)))
+        result = subprocess.check_call(cmake_cmd, stderr=subprocess.STDOUT,
+                                       shell=shell, cwd=str(os.path.join(target_path)))
     except subprocess.CalledProcessError as e:
-        raise GeneratedCodeBuildException('Error occurred during \'cmake\'! More detailed error messages can be found in stdout.')
+        raise GeneratedCodeBuildException(
+            'Error occurred during \'cmake\'! More detailed error messages can be found in stdout.')
```

Comment:
```suggestion
        msg = "Error during 'cmake'. More detailed error messages can be found in stdout."
        raise GeneratedCodeBuildException(msg)
```

Reply:
Apart from the new line after the 79 characters, you still want to escape the single quote in the `cmake`

Reply:
By breaking the line into two statements (`msg = ...` and `raise ...`), none of the lines exceeds the limits and there's no need to escape the single quotes inside the string if you enclose the whole string in double quotes instead.